src/data/move.ts

```diff
@@ -2701,12 +2701,58 @@ export class HitCountPowerAttr extends VariablePowerAttr {
   }
 }
 
+/**
+ * Turning a once was (StatChangeCountPowerAttr) statement and making it available to call for any attribute.
+ * @param {Pokemon} pokemon The pokemon that is being used to calculate the count of positive stats
+ * @returns {number} Returns the amount of positive stats
+ */
+const countPositiveStats = (pokemon: Pokemon): number => {
+  return pokemon.summonData.battleStats.reduce((total, stat) => stat > 0 && stat ? total + stat : total, 0);
```

Comment:
nit: I don't like seeing un-parentheses'd ternary conditions
```suggestion
  return pokemon.summonData.battleStats.reduce((total, stat) => (stat > 0 && stat) ? total + stat : total, 0);
```

Reply:
oh stat should probably be checked first too. `(stat && stat > 0)`